src/meta/model_v2/src/sink.rs

```diff
@@ -127,6 +129,13 @@ impl From<PbSink> for ActiveModel {
             sink_from_name: Set(pb_sink.sink_from_name),
             sink_format_desc: Set(pb_sink.format_desc.as_ref().map(|x| x.into())),
             target_table: Set(pb_sink.target_table.map(|x| x as _)),
+            secret_ref: Set({
```

Comment:
Better to define this column as `json_binary` in migration, then you can define a struct wrapping the map and use it here. Check examples that are using `derive_from_json_struct` for more details.